compass_sdk/compass.py

```diff
@@ -297,7 +297,7 @@ def push_document(
 
         if result.error:
             return result.error
-        return None
```

Comment:
Why was this None before?

Reply:
I don't know, but it shouldn't be None. So I took the chance to fix it